mrp_package_propagation/models/mrp_production.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models, tools
+from odoo.exceptions import UserError
+
+
+class MrpProduction(models.Model):
+    _inherit = "mrp.production"
+
+    is_package_propagated = fields.Boolean(
+        default=False,
+        readonly=True,
+        string="Is package propagated?",
+        help="Package is propagated from a component to the finished product.",
+    )
+    propagated_package_id = fields.Many2one(
+        comodel_name="stock.quant.package",
+        compute="_compute_propagated_package_id",
+        string="Propagated package",
+        help=(
+            "The BoM used on this manufacturing order is set to propagate "
+            "package from one of its components. The value will be "
+            "computed once the corresponding component is selected."
+        ),
+    )
+
+    @api.depends(
+        "move_raw_ids.propagate_package",
+        "move_raw_ids.move_line_ids.qty_done",
```

Comment:
that seems to fail:
`ValueError: Wrong @depends on '_compute_propagated_package_id' (compute method of field mrp.production.propagated_package_id). Dependency field 'qty_done' not found in model stock.move.line.`